mobile/mobile_46.asm

```diff
@@ -2927,6 +2928,7 @@ Unknown_1196b8:
 	db "Sat"
 	db "Sun"
 
+popc
```

Comment:
You should wrap the weekdays here in a setcharmap instead of covering this entire area.

Reply:
Resolved in latest commit.